### ISSUE TITLE
Version 1.5.4 is unusable

### ISSUE THREAD

**Issue by H3llK33p3r** [fonimus/ssh-shell-spring-boot#124]
Hi, 

I create this issue to report an error on the auto configuration part for the tag 1.5.4. 

Here the project compilation 

````
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.483 s <<< FAILURE! - in com.github.fonimus.ssh.shell.basic.DemoApplicationWebTest
[ERROR] testApplicationStartup  Time elapsed: 0.037 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'scriptCommand' defined in URL [jar:file:/C:/workspace/ssh-shell-spring-boot/starter/target/ssh-shell-spring-boot-starter-1.5.4.jar!/com/github/fonimus/ssh/shell/commands/ScriptCommand.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.github.fonimus.ssh.shell.ExtendedShell' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.github.fonimus.ssh.shell.ExtendedShell' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
````

It seems that you need to update the autoconfiguration to return an ExtendedShell instead of a ExtendedShell. 
Moreover there is a cycle injection between the extendedShell and the ScriptCommand. You may been to interface the run or @Lazy the injection inside the scriptCommand ?

The strange things is tyour travis build is ok but me and teammates are unable to use it. 

Another issue is the integration, with classpath resources inside a real jar the process is not able to retrieve the data. 

````
Caused by: java.io.FileNotFoundException: class path resource [pub_keys] cannot be resolved to absolute file path because it does not reside in the file system: jar:file:/C:/m2repo/xxx/xxx/xxx/foo.jat!/pub_keys
	at org.springframework.util.ResourceUtils.getFile(ResourceUtils.java:217) ~[spring-core-5.3.3.jar:5.3.3]
````

Because, the release was not in maven central, I added yours inside our nexus instance. Please this this with a brand new 1.5.5 :) ?

**Comment by fonimus** [comment]
Hi, thanks for the feedback, I opened pull request #128 for both fixes, can you test it and get back to me before I make a new version ?
Thanks

**Comment by H3llK33p3r** [comment]
Still same issue :(. 

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in com.github.fonimus.ssh.shell.commands.ScriptCommand required a bean of type 'com.github.fonimus.ssh.shell.ExtendedShell' that could not be found.
.....................
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   DemoApplicationWebTest.testApplicationStartup » IllegalState Failed to load Ap...
[INFO] 
```

**Comment by fonimus** [comment]
Fixed by merge pr #128
